openkfw/TruBudget
# 1781 Remove breadcrumbs from the navigation and move them before main content

### Checklist




### Description
- renamed MainNavbarNavigation to Breadcrumbs
- move around redux logic so Breadcrumbs can be used in Main component below the navigation itself
- changed styles accordingly because Breadcrumbs changed its place and they are now below navigation and before main content

<img width="688" alt="Screenshot 2024-04-18 at 08 33 23" src="https://github.com/openkfw/TruBudget/assets/55734106/e27abf7a-d6e5-4a08-83a5-973f1aed9455">
<img width="685" alt="Screenshot 2024-04-18 at 08 43 36" src="https://github.com/openkfw/TruBudget/assets/55734106/f73756fc-2e28-40f4-b6d0-63c5325fc46d">

### How to test

1. login as any user
2. go to projects -> subprojects -> workflow items and see that breadcrumbs position has changed and right side navigation buttons are not hidden below breadcrumbs as before
3. try to click on any breadcrumb so see if you are redirected properly
4. go to developer tools and change width of the window to 768px and change to Card view in /projects
5. click on search icon and see that search bar is completely visible (before it was partially hidden below the breadcrumbs)

Closes #1781
